lms/djangoapps/teams/views.py

```diff
@@ -338,13 +338,20 @@ def get(self, request):
                 size=MAXIMUM_SEARCH_SIZE,
             )
 
+            page = self.get_page()
```

Comment:
I had used `page` in a few places temporarily, and failed to unname it once those usages were gone.